kalitectl.py

```diff
@@ -480,12 +454,13 @@ def start(debug=False, daemonize=True, args=[], skip_job_scheduler=False, port=N
     if daemonize:
         
         from django.utils.daemonize import become_daemon
-        logfile = os.path.join(KALITE_HOME, "kalite.log") if os.environ.get("NAIVE_LOGGING", False) else None
-        if logfile:
-            print("With logging...")
-            become_daemon(out_log=logfile, err_log=logfile)
-        else:
-            become_daemon()
+        kwargs = {}
+        # Truncate the file
+        open(SERVER_LOG, "w").truncate()
+        print("Going to daemon mode, logging to {0}".format(SERVER_LOG))
+        kwargs['out_log'] = SERVER_LOG
+        kwargs['err_log'] = SERVER_LOG
```

Comment:
Sounds like an interesting error, could you post a separate issue?

It *could* be because that the stream isn't threadsafe... logging writes to stdout and then that's actually redirected to our file stream. It might not happen with DEBUG=False, which is the default... which were you testing with?